utils/node_local_test.py

```diff
@@ -453,18 +453,18 @@ def start(self):
         while True:
             time.sleep(0.5)
             rc = self.run_dmg(cmd)
+            if rc.returncode == 0:
+                break
             ready = False
-            if rc.returncode == 1:
-                for line in rc.stdout.decode('utf-8').splitlines():
-                    if 'format storage of running instance' in line:
-                        ready = True
-                    format_message = ('format request for already-formatted'
-                                      ' storage and reformat not specified')
-                    if format_message in line:
-                        cmd = ['storage', 'format', '--reformat']
-                for line in rc.stderr.decode('utf-8').splitlines():
-                    if 'system reformat requires the following' in line:
-                        ready = True
+            for line in rc.stderr.decode('utf-8').splitlines():
+                if 'format storage of running instance' in line:
+                    ready = True
+                format_message = ('format request for already-formatted'
+                                  ' storage and reformat not specified')
+                if format_message in line:
+                    cmd = ['storage', 'format', '--reformat']
+                if 'system reformat requires the following' in line:
+                    ready = True
```

Comment:
You could probably tidy this up a bit by moving it around slightly but overall a big improvement. 
```suggestion
                if 'format storage of running instance' in line or \
                    'system reformat requires the following' in line:
                    ready = True
                    break
                format_message = ('format request for already-formatted'
                                  ' storage and reformat not specified')
                if format_message in line:
                    cmd = ['storage', 'format', '--reformat']
```